lib/matplotlib/axes/_secondary_axes.py

```diff
@@ -275,6 +300,14 @@ def set_color(self, color):
     See :doc:`/gallery/subplots_axes_and_figures/secondary_axis`
     for examples of making these conversions.
 
+transform : `.Transform`, optional
+    An optional transform that can be passed. *location* will be
```

Comment:
The first sentence is somewhat redundant.
```suggestion
    If specified, *location* will be
```